components/paragraph/src/link-renderer.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import Anchor from './atoms/anchor';
+
+// Support react-router links
+// https://github.com/rexxars/react-markdown/issues/29#issuecomment-231556543
+function LinkRenderer({ href, children }) {
```

Comment:
as a functional component it's probably more conventional for this to be an arrow function :grin:

Reply:
Great shout, thank you!